packages/doc/content/components/components/popover/index.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: 'Popover'
+metaTitle: 'Popover'
+metaDescription: 'Popover Component'
+---
+
+# Popover
+
+### Reference
+
+Gympass `<Popover />` description
+
+### Usage
```

Comment:
I still would split the basic usage as you did in last commit, and another section explaining the position prop and its results as you did here

like [Mui does](https://mui.com/material-ui/react-tooltip/#positioned-tooltips)

Reply:
Done!